site/pages/docs/ref/filter/filter-en.hbs

```diff
@@ -48,6 +48,16 @@
 				<dd>Otherwise specified, it will filter and search on <code>&lt;li&gt;</code> or on <code>&lt;tr&gt;</code> in <code>&lt;tbody&gt;</code> for tables</dd>
 			</dl>
 		</li>
+		<li>
+			<p><strong>Optional:</strong> Override the default search type to perform an "AND" or an "OR" search by passing a JSON array through the <code>data-wb-filter</code> attribute and setting the <code>filterType</code> parameter.</p>
```

Comment:
Consider writing all instances of the ``and`` and ``or`` operators consistently. Currently they're written in multiple ways throughout the content. For example, the ``or`` operator is currently mentioned in several different ways:

* OR
* "OR"
* ``OR``
* ``or``
* "or"